staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

```diff
@@ -89,7 +97,7 @@ func NewCmdExplain(parent string, f cmdutil.Factory, streams genericiooptions.IO
 	o := NewExplainOptions(parent, streams)
 
 	cmd := &cobra.Command{
-		Use:                   "explain RESOURCE",
+		Use:                   "explain TYPE [--recursive=FALSE|TRUE] [--api-version=api-version-group] [--output=plaintext|plaintext-openapiv2|plaintext-openapiv3]",
```

Comment:
I tried the `DisableFlagsInUseLine: false`
However the mention of flag is like this:

`kubectl explain TYPE [flags] [options]`

I think we can keep it true and set Use as 
`explain TYPE [--recursive=FALSE|TRUE] [--api-version=api-version-group] [--output=plaintext|plaintext-openapiv2|plaintext-openapiv3]`

Reply:
OK Thanks for investigating this, it's better this way.

Reply:
I think you should keep "resource" though, as confusing as it is, it's actually a resource.

Reply:
I was planning to write Resource as it is delivering clarity . However just to verify i checked with official syntax in [kubectl Cheatsheet](https://kubernetes.io/docs/reference/kubectl/#syntax) , we can see for `resource` TYPE is suggested for all help output

> TYPE: Specifies the [resource type](https://kubernetes.io/docs/reference/kubectl/#resource-types). Resource types are case-insensitive and you can specify the singular, plural, or abbreviated forms. For example, the following commands produce the same output:

WDYT? should we change it. I like RESOURCE more though idk if this will cause some syntax or style issue with official statements.

Reply:
One more update I checked the command with `--output plaintext-openapiv3`  and it didn't work so removing it for now